rllib/agents/dqn/tests/test_apex.py

```diff
@@ -14,19 +14,29 @@ def setUp(self):
     def tearDown(self):
         ray.shutdown()
 
-    def test_apex_epsilon_distribution(self):
+    def test_apex_compilation_and_per_worker_epsilon_values(self):
```

Comment:
Merged test cases for APEX compilation (+ train() calls) and epsilon value (per-worker) checks.

Reply:
to save some time.